packages/synapse-react-client/src/components/OAuthClientManagement/OAuthManagement.tsx

```diff
@@ -57,6 +58,7 @@ export const OAuthManagement: React.FunctionComponent = () => {
           setIsEdit(false)
         }}
         sx={{ float: 'right' }}
+        startIcon={<AddCircleTwoTone />}
```

Comment:
Small changes to match design -- add an icon to the button, button text should not be allcaps